docs/RFCS/20170925_jsonb_scope.md

```diff
@@ -0,0 +1,123 @@
+- Feature Name: jsonb_scope
+- Status: draft
+- Start Date: 2017-09-25
+- Authors: Masha Schneider, Justin Jaffray
+- RFC PR: ...
+- Cockroach Issue: [#2969](https://github.com/cockroachdb/cockroach/issues/2969)
+
+# Summary
+
+Support JSONB in CockroachDB.
+JSONB is a column type supported by Postgres that allows users to store and
+manipulate JSON documents directly in their database.
+It's distinguished from Postgres's plain `JSON` column type in that it stores
+a binary representation of the documents, rather than the raw JSON string.
+The high-level overview of the features we intend to include is:
+* JSON Encoding/Decoding
+* Key Encoding for JSON values
```

Comment:
highly optional nit: these first two bullets should use sentence case like the following bullets